src/compas_fab/robots/robot.py

```diff
@@ -453,21 +403,53 @@ def get_configurable_joint_types(self, group=None):
         configurable_joints = self.get_configurable_joints(group)
         return [j.type for j in configurable_joints]
 
+    def get_links_distance(self, configuration, link_name_1, link_name_2, group=None):
```

Comment:
similar here, this is a convenience function, of which I don't see the point of adding to the library.